bool on_duty
int32 vehicle_id
sring service_station_id
float32 x
float32 y
